js/index.d.ts

```diff
@@ -1,47 +1,159 @@
-/*
-  Typings for @sentry/cli
-*/
+/**
+ * Typings for @sentry/cli
+ */
 declare module '@sentry/cli' {
   export interface SentryCliOptions {
+    /**
+     * Sentry instance
+     */
     url?: string;
+    /**
+     * Authentication token for API, interchangeable with `apiKey`
+     */
     authToken?: string;
+    /**
+     * Authentication token for API, interchangeable with `authToken`
+     */
     apiKey?: string;
+    /**
+     * Sentry DSN
+     */
     dsn?: string;
+    /**
+     * Organization slug
+     */
     org?: string;
+    /**
+     * Project Project slug
+     */
     project?: string;
+    /**
+     * VCS remote name
+     */
     vscRemote?: string;
+    /**
+     * Unique identifier for the distribution, used to further segment your release.
+     * Usually your build number.
+     */
+    dist?: string;
+    /**
+     * If true, all logs are suppressed
+     */
     silent?: boolean;
   }
 
   export interface SentryCliUploadSourceMapsOptions {
+    /**
+   * One or more paths that Sentry CLI should scan recursively for sources.
+   * It will upload all .map files and match associated .js files.
+   */
```

Comment:
```suggestion
    /**
     * One or more paths that Sentry CLI should scan recursively for sources.
     * It will upload all .map files and match associated .js files.
     */
```